test/workers/hermes_worker.js

```diff
@@ -55,10 +65,11 @@ describe('Hermes Worker', () => {
     mockDataModelEngine.initialiseBundling.resolves(mockResult);
     mockUploadRepository.bundleSizeLimit.resolves(bundleSizeLimit);
     await hermesWorker.beforeWorkLoop();
+    await hermesWorker.work();
```

Comment:
You shouldn't call work in tests. It means the worker starts calling the `periodicWork` method rapidly, and we loose the ability to control the execution time of the actual method under test.